README.md

```diff
@@ -133,15 +133,15 @@ console.log(state.data.getConfig());
 Before you can collect other checkout information from the customer, you should first ask them to sign in. Once they are signed in, the checkout state will be populated with their personal details, such as their addresses.
 
 ```js
-const state = await service.signInCustomer('foo@bar.com', 'password123');
+const state = await service.signInCustomer({email: 'foo@bar.com', password: 'password123'});
```

Comment:
Thanks @fresh5447. Yes, you're right, the example needs to be updated! 😄Can you please add a space after `{` and before `}`?

Reply:
No problem - @davidchin  change made.

I also just committed one more change. I was only able to get the method `selectShippingOption` by passing it *only* the selected option ID. So I assume the docs are outdated here as well. Furthermore, it doesn't appear an address has an id when you query `getShippingAddress`